src/main/java/frc/trigon/robot/subsystems/arm/ArmCommands.java

```diff
@@ -0,0 +1,45 @@
+package frc.trigon.robot.subsystems.arm;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.FunctionalCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class ArmCommands {
+    private static final Arm ARM = Arm.getInstance();
+
+    public static Command setTargetState(Rotation2d targetAngle, double targetElevatorMeters) {
```

Comment:
correct names

Reply:
yishai told me call it like that

Reply:
What's a target elevator? And add what the function returns in the name